Cargo.toml

```diff
@@ -9,38 +9,24 @@ edition = "2018"
 build="build/build.rs"
 
 [dependencies]
-curve25519-dalek = { version = "2", features = ["serde", "avx2_backend"] }
-merlin = "2"
-lazy_static = "1.3.0"
+bls12_381 = { git = "https://github.com/zkcrypto/bls12_381", branch = "master" }
+lazy_static = "1.3"
 
-[dependencies.bulletproofs]
-git = "https://github.com/dusk-network/bulletproofs"
-branch = "develop"
-features = ["yoloproofs"]
+[dependencies.plonk]
+git = "https://github.com/dusk-network/plonk"
+branch = "vlopes11/stable"
```

Comment:
As discussed, change this to `branch = "stable"`.
Basically the [Plonk PR 126](https://github.com/dusk-network/plonk/pull/126) needs to be landed first.